libraries/src/Form/Field/ImagelistField.php

```diff
@@ -3,43 +3,61 @@
 /**
  * Joomla! Content Management System
  *
- * @copyright  (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
- * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ * @copyright (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
  */
 
 namespace Joomla\CMS\Form\Field;
 
+use Joomla\CMS\Component\ComponentHelper;
+
 // phpcs:disable PSR1.Files.SideEffects
-\defined('JPATH_PLATFORM') or die;
+\defined("JPATH_PLATFORM") or die();
```

Comment:
```suggestion
\defined('JPATH_PLATFORM') or die;
```
Please revert your change or accept my suggestion which does the same. You should not make unrelated changes to code style.